### PR TITLE
Fix small issues concerning namespacing in test-all suite

### PR DESCRIPTION
**Pull request by luke-gru**
Fix temporary methods on Object leaking across test cases.

### CODE REVIEW

File: test/irb/yamatanooroti/test_rendering.rb [hunk 1]
```diff
@@ -2,248 +2,249 @@
 
 begin
   require 'yamatanooroti'
+rescue LoadError, NameError
```
Comment (luke-gru) [inline test/irb/yamatanooroti/test_rendering.rb:5]: The diff for this file without whitespace is just this rescue block

**Comment by luke-gru** [comment]
For the IRB test file, I wasn't sure if I needed to modify it in the IRB repo, or if changes could go in both directions (that is, can be modified here first and then the maintainer cherry-picks the changes). If it's the first one, then I can re-push without the IRB changes.

**Review by hsbt — CHANGES_REQUESTED** [review]
Can you submit `irb` tests to https://github.com/ruby/irb?

**Comment by luke-gru** [comment]
Yes I will submit those changes separately to IRB. I've updated the branch 👍

**Review by hsbt — APPROVED** [review]

**Comment by hsbt** [comment]
@luke-gru Thanks!

**Pull request merged**